prediction_market_agent/agents/microchain_agent/microchain_agent.py

```diff
@@ -107,7 +106,9 @@ def main(
 ) -> None:
     # This description below serves to unique identify agent entries on the LTM, and should be
     # unique across instances (i.e. markets).
-    unique_task_description = AgentIdentifier.microchain_task_from_market(market_type)
+    unique_task_description = (
```

Comment:
So that running locally doesn't add memories under the same description as deployed agent